app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/model/NotificationType.kt

```diff
@@ -34,14 +35,14 @@ enum class NotificationType(@StringRes val type: Int) {
             return TEN_MINUTES
         }
 
-        fun remindAt(time: LocalDateTime, notificationType: NotificationType): LocalDateTime {
+        fun remindAt(time: LocalDateTime, notificationType: NotificationType): Long {
             return when (notificationType) {
                 TEN_MINUTES -> time.minusMinutes(10)
                 THIRTY_MINUTES -> time.minusMinutes(30)
                 ONE_HOUR -> time.minusHours(1)
                 SIX_HOURS -> time.minusHours(6)
                 ONE_DAY -> time.minusDays(1)
-            }
+            }.toLong()
```

Comment:
You need to be careful that if you either pass in a UTC time for the `LocalDateTime` instance or you save the time zone information together with this timestamp. Otherwise, you might get wrong times when the user travels